<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ui_ImageOptions</class>
 <widget class="QWidget" name="Ui_ImageOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>24</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>22</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>32767</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Insert Options</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QHBoxLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QLabel" name="lAngle">
     <property name="text">
      <string>Angle:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leAngle">
     <property name="toolTip">
      <string>Rotation Angle</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lFactor">
     <property name="text">
      <string>Factor:</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leFactor">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Scale Factor</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lDPI">
     <property name="text">
      <string>DPI</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leDPI">
     <property name="toolTip">
      <string>DPI of Image</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="sep1_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::VLine</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>leAngle</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Ui_ImageOptions</receiver>
   <slot>updateData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leDPI</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Ui_ImageOptions</receiver>
   <slot>updateDPI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leFactor</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Ui_ImageOptions</receiver>
   <slot>updateFactor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>6</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateFactor()</slot>
 </slots>
</ui>
